components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,54 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File to TMP",
+  description: "Download a specific file to the temporary directory. [See the documentation](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor).",
```

Comment:
```suggestion
  description: "Download a specific file to the temporary directory. [See the documentation](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)."
```

Reply:
Changed "See docs here" to "See the documentation" to follow Google's Developer docs style guide.